src/translib/transformer/xfmr_intf.go

```diff
@@ -916,25 +923,12 @@ func convertIpMapToOC (intfIpMap map[string]db.Value, ifInfo *ocbinds.Openconfig
     return err
 }
 
-func validateIPexist(intTbl IntfTblData, inParams *XfmrParams, intfName *string, checkExists *bool) error {
-    intfIPKeys, _ := inParams.d.GetKeys(&db.TableSpec{Name:intTbl.cfgDb.intfTN})
-    log.Info("intfIPKeys: ", intfIPKeys)
-    subOpMap := make(map[db.DBNum]map[string]map[string]db.Value)
-    resMap := make(map[string]map[string]db.Value)
+func interfaceIPcount(tblName string, d *db.DB, intfName *string, ipCnt *int) error {
+    intfIPKeys, _ := d.GetKeys(&db.TableSpec{Name:tblName})
     if len(intfIPKeys) > 0 {
-        *checkExists = false
-        /* Check if IP entries exist for given Interface*/
         for i := range intfIPKeys {
             if *intfName == intfIPKeys[i].Get(0) {
-                *checkExists = true
-                if len(intfIPKeys[i].Comp) > 1 {
-                    //Keep subOpDataMap[DELETE] as empty 
-                    subOpMap[db.ConfigDB] = resMap
-                    inParams.subOpDataMap[DELETE] = &subOpMap
-                    errStr := "Need to first remove the IP configuration"
-                    log.Error(errStr)
-                    return errors.New(errStr)
-                }
+                *ipCnt = *ipCnt+1
```

Comment:
Are we counting here just the IP address entries or entries with IP and entry with ifname together?